pyproject.toml

```diff
@@ -66,7 +66,7 @@ all = [
     "aiohttp <4.0"
 ]
 lab = [
-    "imjoy-jupyterlab-extension >=0.1.13",
+    "imjoy-jupyterlab-extension ==0.1.13",
```

Comment:
What's the reason to pin it to the old version? I thought we need that for the new jupyterlab. What did I missed?

Reply:
I changed to follow this:
https://github.com/InsightSoftwareConsortium/itkwidgets/pull/689/commits/94fee5711ec87478d902a4c4b87047b08209ce48

Which references this issue that I have not checked myself:
https://github.com/imjoy-team/imjoy-jupyterlab-extension/issues/4